packages/sd-jwt/src/verify.ts

```diff
@@ -0,0 +1,117 @@
+import * as jwt from "jsonwebtoken";
+import * as jose from "jose";
```

Comment:
Why do we use two different libraries to handle JWTs?

Reply:
`jsonwebtoken` uses `jose` under the hood, I thought it was simple to just use it as well instead of relying on `jsonwebtoken` API
(`jose` is still a dependency to us)

Reply:
I double-checked and I guess I was wrong: `jsonwebtoken` uses `jws` not `jose`: https://github.com/auth0/node-jsonwebtoken/blob/a99fd4b473e257c2f50ff69c716db1c520bf9a78/decode.js#L1

It's `jwks-rsa` (another dependency of ours) that depends on `jose`. Thus the point of the previous comment.

I will check if it makes sense to not use `jose` directly.

Reply:
Done in https://github.com/pagopa/io-wallet/pull/4/commits/8aebe035174a823268ed5f4d973abe57b611a1e7

Thanks!